diff --git a/roles/openshift_facts/library/openshift_facts.py b/roles/openshift_facts/library/openshift_facts.py
index aff7d72..cfe7b38 100755
--- a/roles/openshift_facts/library/openshift_facts.py
+++ b/roles/openshift_facts/library/openshift_facts.py
@@ -298,7 +298,8 @@ def set_fluentd_facts_if_unset(facts):
             missing
     """
     if 'common' in facts:
-        deployment_type = facts['common']['deployment_type']
+        # deployment_type = facts['common']['deployment_type']
+        deployment_type = 'enterprise'
         if 'use_fluentd' not in facts['common']:
             use_fluentd = True if deployment_type == 'online' else False
             facts['common']['use_fluentd'] = use_fluentd
